auraed/src/ebpf/loader.rs

```diff
@@ -136,7 +166,7 @@ impl BpfLoader {
 
                     if events.lost > 0 {
                         warn!(
-                            "queues are getting full, lost {} perf events",
+                            "queues are getting full, dropped {} perf events",
```

Comment:
Can we please open up an issue for dropped eBPF events here? I believe we will need to start understanding the conditions in which something like this will happen and see if we can't find ways of getting closer to a guarantee that we do not drop events.

Reply:
There is only one condition in which this will happen and this is when our user-space buffer is not large enough to hold all the events we read from the per-CPU buffer: https://github.com/JeroenSoeters/aurae/blob/sensible-buffer-sizes/auraed/src/ebpf/loader.rs#L156-L157

`.read_events()` will store in the buffer whatever fits and everything that does not fit is "dropped"

Reply:
My quick review is telling me we don’t need to be concerned about this condition in that case — should we panic in the event of a drop? Or at least increase the log level?

Reply:
Actually... rereading my own code :) this should never happen anymore as those buffers are now the same size as I pass the buffer size now also to `perf_array.open()`. Maybe we should make this an error instead of a warning? 🤔

Reply:
Heh we had the same conclusion - yes let’s go with error level

Reply:
https://github.com/JeroenSoeters/aurae/blob/sensible-buffer-sizes/auraed/src/ebpf/loader.rs#L139-L140

Reply:
I've changed all the warnings to errors, as they all really shouldn't happen.